Makefile

```diff
@@ -30,7 +30,7 @@ endif
 endif
 
 # Determine which qmk cli to use
-ifeq (, $(shell which qmk))
+ifeq ($(shell which qmk),)
```

Comment:
I think the culprit is the extra whitespace here. I'd prefer to narrowly fix this than change the other behavior just yet.